src/api-reference/invoice/v3.purchase-order-receipt.markdown

```diff
@@ -0,0 +1,180 @@
+---
+title: Purchase Order Receipt v3
+layout: reference
+---
+
+* [Create a new purchase order receipt](#post)
+* [Update purchase order line item with receipt information](#put-receipts)
+* [Get existing purchase order receipt](#get)
+* [Delete existing purchase order receipt](#delete)
+* [Schema](#schema-receipt)
+* [Response schema](#schema-response)
+* [Receipt schema](#schema-receipt)
+* [Error codes](#error-codes)
+
+## Version
+
+3.0  
+
+## <a name="post"></a>Create a new purchase order receipt 
+
+```
+POST /api/v3.0/invoice/purchaseordersreceipts
+```
+
+Creates purchase order receipts and associate if to purchase order line item and returns status of updation.
```

Comment:
Suggested text: Creates purchase order receipt, associates it to purchase order line item, and returns update status.

Reply:
Fixed in bbd8fe90